Modix/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -147,11 +148,13 @@ public static IServiceCollection AddModix(this IServiceCollection services)
             return services;
         }
 
-        public static IServiceCollection AddStatsD(this IServiceCollection services, IHostingEnvironment environment)
+        public static IServiceCollection AddStatsD(this IServiceCollection services, IHostingEnvironment environment, IConfiguration configuration)
         {
             var cfg = new StatsdConfig { Prefix = "modix" };
 
-            if (!environment.IsProduction() && string.IsNullOrWhiteSpace(cfg.StatsdServerName))
+            var enableStatsd = configuration.GetValue<bool>(nameof(ModixConfig.EnableStatsd));
+            if (!enableStatsd ||
+                environment.IsDevelopment() && string.IsNullOrWhiteSpace(cfg.StatsdServerName))
```

Comment:
This should be switched back to `!environment.IsProduction()`

Reply:
This is done.